sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1720,4 +1720,11 @@ object functions {
      UnresolvedFunction(udfName, cols.map(_.expr))
   }
 
+  /**
+   * Parses the comma separated expression strings into the columns they represent, similar to
+   * DataFrame.selectExpr
+   *
+   * @group normal_funcs
+   */
+  def expr(expr: String): Column = Column(new SqlParser().parseExpression(expr))
```

Comment:
one thing - can you move this higher up? maybe to the first block of functions. in general i don't think we put functions at the end like this, after all the callUDFs